### ISSUE TITLE
The header options are not visible and does not receive visible focus after enabling the sidekick extension 

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#252]
**Description**
Enabling the sidekick extension hides the top header options and the magnifier icon. These options are still available to a screen reader user.

**To Reproduce**
1. Go to URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Press Alt key to go to the chrome menu and press tab to go to the sidekick extension or enable sidekick extension using mouse.
3. Verify if the top header options are still visible.

**Expected behavior**
The top header options should be visible even after enabling the extension as the items in the header are accessible to screen reader users. 

**Screenshots**
The screenshot is attached which shows the screen reader accessing the hidden menus.
![Top header not visible after adding sidekick extension but screen reader can access it](https://user-images.githubusercontent.com/123964084/218763552-243fed80-d7e9-488d-bd49-4848ab9b9c9a.png)
 
**Additional context**
https://www.w3.org/TR/WCAG21/#focus-visible

**Why it matters?**
Keyboard users need to know which component currently has the input focus so they can predict the results of their key presses. By default, web browsers indicate focus visually, but custom programming, styling, and scripting can disrupt it.

**Comment by rofe** [comment]
@puja-iaccessible: by default, the sidekick doesn't push down content as it can lead to layout complications on certain customer websites or within editors (Word Online, Google Docs). If preferred, this setting can be changed under Options > User experience.

**Comment by rofe** [comment]
Closing this as won't fix: users can configure their sidekick to push down content (this setting can also be enforced at the project wide level).
